bin/release_version.sh

```diff
@@ -61,7 +52,7 @@ FRAMEWORK_NAME=Purchases.framework
 
 mv $FRAMEWORK_NAME.zip $CARTHAGE_UPLOADS_PATH
 
-fastlane github_release $CURRENT_VERSION
+fastlane ios github_release version:$CURRENT_VERSION
```

Comment:
what was the previous behavior?

Reply:
it worked correctly without it, but I think that it working or not depends on the fastlane version